content/en-us/luau/functions.md

```diff
@@ -93,6 +93,10 @@ secondPart:Destroy()
 print(secondPart.Parent) -- nil
 ```
 
+<Alert severity = 'info'>
+The statement <code>table:Method()</code> is functionally identical to <code>table.Method(table)</code>, and when defining a method, <code>function table:Method()</code> is identical to <code>function table.Method(self)</code>. This behavior extends to Instance methods, since they are also derived from tables.
+</Alert>
```

Comment:
```suggestion
<Alert severity = 'info'>
The statement `table:Method()` is functionally identical to `table.Method(table)`, and when defining a method, `function table:Method()` is identical to `function table.Method(self)`. This behavior extends to `Class.Instance` methods, since they are also derived from tables.
</Alert>
```

Reply:
It wasn't rendering in my preview tool but I guess Roblox's parser handles it 😅